test/server/getters/get-fee-window-current.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
```

Comment:
This file doesn't seem to test getFeeWindowCurrent in any way...

Reply:
uhhhh, yea.

Reply:
Weird, there was a race between my IDE writing the file out and the git add

Reply:
fixed now